src/main/java/Duke.java

```diff
@@ -2,11 +2,19 @@
 import java.util.Arrays;
 
 public class Duke {
+    //List of Special User Commands
+    public static final String EXIT_STRING = "bye";
+    public static final String LIST_COMMAND = "list";
+    public static final String DONE_COMMAND = "done";
+    public static final String TODO_COMMAND = "todo";
+    public static final String DEADLINE_COMMAND = "deadline";
+    public static final String EVENT_COMMAND = "event";
```

Comment:
It is quite good practice to define the command using constant, which will greatly reduce time needed to modify the code once there is a change to be made.